### PR TITLE
fix: Joining with an invalid version

### PR DESCRIPTION
**Pull request by SohKey**
fix: #396

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -59,6 +59,24 @@ export function create_server() {
   const mobs_position = mobs_position_factory(world)
 
   server.on('login', client => {
+    const {
+      // @ts-ignore
+      mcversion: { minecraftVersion, version },
```
Comment (SohKey) [inline src/server.js:64]: I dont know why `mcversion` is not recognized, i'm forced to use ts-ignore

**Review by Sceat — CHANGES_REQUESTED** [review]
This issue is more about fixing the problem in prismarine or wherever it is, and allowing players to join regardless of their minor versions

**Comment by Sceat** [comment]
@SohKey I guess you can make some research work in here, but if it's not fixable on AresRPG, this pr should be closed

**Comment by Sceat** [comment]
After some thinking, this is actually okay to have as a temporary fix. @SohKey can you just improve the `end` message similarly to https://github.com/aresrpg/aresrpg/blob/master/src/player/resource_pack.js

Note that later on we'll need to find a way to support multiple version of the same protocol

**Review by Sceat — APPROVED** [review]
lgtm

**Review by Sceat — APPROVED** [review]

**Pull request merged**